### ISSUE TITLE
Sections 2.7.4.2 and 2.11.5 - Nonfunctional Figures

### ISSUE THREAD

**Issue by dfailing** [katharynduffy/Environmental-Informatics-Using-Research-Infrastructures-and-their-Data#19]
A couple of the figures (for the File browse pane in RStudio showing .Renviron file." and the "example of camera data (DP1.30010.001) from the San Joaquin...") are not functioning - instead of the image, they are revealing the code between the <figure> </figure> tags. The actual image files are accessible (one on NEON's github, one on the local site), so I'm not sure why they aren't working, since there are no functional <figure> environments in the first two chapters.

**Comment by katharynduffy** [comment]
Awesome, thanks for catching that David!

**Comment by katharynduffy** [comment]
@dfailing , do you mean the NEON chapter or the PhenoCam chapter?

**Comment by dfailing** [comment]
The NEON chapter, and the two sections I referenced in the title of this Issue - Sections 2.7.4.2 and 2.11.5. I wonder if anyone else is having this issue, or if it's my browser not rendering things properly. I attempted to attach screenshots to this comment of what I'm seeing.

![Screen Shot 2020-08-19 at 10 05 25 AM](https://user-images.githubusercontent.com/54870429/90667430-9e7d1c00-e203-11ea-80a2-33f650426ef2.png)
![Screen Shot 2020-08-19 at 10 05 57 AM](https://user-images.githubusercontent.com/54870429/90667432-9fae4900-e203-11ea-85c4-003e0c49d8e9.png)
